README.md

```diff
@@ -38,7 +40,9 @@ Main function currently shows an example of how to use this project. To write a
 
 ## Notes
 
-* Setting thread affinity is not supported on MacOS (yet?). As such, `getAffinityManager()` returns `null` on MacOS. 
-* Currently, there is no CLI or any other defined interface. The `main()` function is quite messy and likely will remain so (at least for now).
-* For some reason, running a lot of different tests in one go will drastically reduce the performance and weak outcomes frequency. For now, better avoid running for longer than 5 minutes.
+* Setting thread affinity is not supported on macOS (yet?). As such, `getAffinityManager()` returns `null` on macOS.
+* Currently, there is no CLI or any other defined interface. The `main()` function is quite messy and likely will remain
+  so (at least for now).
+* For some reason, running a lot of different tests in one go will drastically reduce the performance and weak outcomes'
+  frequency. For now, better avoid running for longer than 5 minutes.
```

Comment:
Are there any substantial changes in `README.md`?
If not I would propose to revert the changes to keep the diff shorter.

Reply:
MacOS -> macOS (because IDEA told so)